functions/Get-DbaSqlService.ps1

```diff
@@ -65,57 +68,38 @@ Function Get-DbaSqlService {
 	Param (
 		[parameter(ValueFromPipeline = $true)]
 		[Alias("cn", "host", "Server")]
-		[string[]]$ComputerName = $env:COMPUTERNAME,
+		[DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
 		[Alias("Instance")]
 		[string[]]$InstanceName,
 		[PSCredential]$Credential,
-		[ValidateSet("Agent", "Browser", "Engine", "FullText", "SSAS", "SSIS", "SSRS", "AnalysisServer", "ReportServer", "Search", "SqlAgent", "SqlBrowser", "SqlServer", "SqlServerIntegrationService")]
+		[ValidateSet("Agent", "Browser", "Engine", "FullText", "SSAS", "SSIS", "SSRS", "AnalysisServer", "ReportServer", "FullTextFilter Daemon Launcher", "FullText Search", "SqlAgent", "SqlBrowser", "SqlServer", "SqlServerIntegrationService")]
```

Comment:
There is already code break...the command is in a previous PR to be rewritten already. We need to pick one because you have to tab 15 times to get to `SqlServerIntegrationService`. I would go for the short names and see no reason to have it standardized to a corresponding SMO object.

I believe the original purpose of the shorter names was to map those to the publicly-known name the services are referenced by.